tool/ide/RustManifestSyncer.kt

```diff
@@ -141,14 +163,16 @@ class RustManifestSyncer : Callable<Unit> {
             val cargoToml = Config.inMemory();
             val subConfig = cargoToml.createSubConfig()
             subConfig.set<List<String>>("members", manifestPaths)
+            subConfig.set<String>("resolver", "2")
             cargoToml.set<Config>("workspace", subConfig)
-            cargoToml.set<String>("resolver", "2")
             return cargoToml
         }
 
         private fun loadSyncProperties(bazelBin: File): List<TargetProperties> {
             return findSyncPropertiesFiles(bazelBin)
-                    .map { TargetProperties.fromPropertiesFile(it) }
+                    .map { TargetProperties.fromPropertiesFile(it, workspaceRefs) }
+                    .groupBy { Pair(it.name, it.path) }.values
+                    .map { TargetProperties.mergeList(it) }
```

Comment:
Merging e.g. `typedb-driver` and `typedb-driver-sync` into one crate with optional feature `sync` (specified in correpsonding `BUILD`)